packages/cli/src/commands/process/create.ts

```diff
@@ -36,6 +37,23 @@ export default class ProcessCreate extends Command {
     if (hashes.length != 1) throw new Error('error while getting the hash of the process created')
     const hash = hashes[0]
     this.spinner.stop(hash)
-    return this.lcd.process.get(hash)
+    const process = await this.lcd.process.get(hash)
+    this.createTransfer(process, mnemonic)
+    return process
+  }
+
+  async createTransfer(process: IProcess, mnemonic: string) {
+    const acc = await this.lcd.account.import(mnemonic)
+    this.spinner.start('Transfer credits to the process')
```

Comment:
why need to use only one word for coins/tokens/credits. The rest of the commands are currently using "tokens"
```suggestion
    this.spinner.start('Transfering tokens to the process')
```